### DISCUSSION TITLE
Waiting too long for GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by Kubo252** [community/community#120272] [GitHub Education]
### Select Topic Area

Question

### Body

I have rights to use Copilot through my university. Firstly my application was denied (probably because the photo of academic affiliation documents was not great). I have submitted my application again, but now I am waiting around 2 months. Can you please help me to get an access? 
Thank you very much

<img width="262" alt="access denied" src="https://github.com/community/community/assets/102022474/9b6de400-2af3-4a80-99f8-0fe3df420ae9">
<img width="1031" alt="waiting" src="https://github.com/community/community/assets/102022474/7f727952-e6e2-475c-9de6-cc9bace1412c">

**Comment by cryptofino** [comment]
respoens*1
